docs/tutorials/gettingstartedexample/gse-gen-common.md

```diff
@@ -28,17 +28,267 @@ This document covers a range of common scenarios using the PXIe-148X Generation
+The logic for FPGA display parameters is shared for all PXIe-148X getting started examples. Therefore, the acquisition tutorial for this topic is used with a few minor differences.
+
+1. Complete the steps in [PXIe-148X Getting Started Example - Common Acquisition Tutorials](gse-acq-common.md#reducing-system-bandwidth-usage) with the following differences.
+    - When running the Generation Example VI, wait for the **Waiting for Serializer Setup** indicator to illuminate then click **Serializer Setup Complete**.
 
 ## Setting RAW Display Parameters
-TODO: Reference the equivalent Acquisition section
+
+This tutorial shows you how to configure the **RAW Display Parameters** to change the interpretation of images being displayed during generation. The logic for FPGA display parameters is shared for all PXIe-148X getting started examples. Therefore, the acquisition tutorial for this topic is used with a few minor differences.
+
+> Note: If using a TDMS file created with the Create CSI-2 Packet TDMS Files utility, the impact of the RAW display parameter settings will differ from a TDMS file containing image data acquired from a camera.
+
+1. Set the following controls on the Generation Example GSE VI and leave all other values at their defaults.
+    > Note: VI controls and indicators can be reset to default values by clicking on the **Edit** menu and selecting the **Reinitialize Values to Default** option.
+
+    | Tab | Control | Value |
+    |---|---|---|
+    | Resource | RIO Device | [System Specific] |
+    | Resource | Bitfile Path | [Refer to Bitfile Path in the PXIe-148X Generation GSE Help](../../reference/gettingstartedexample/gse-gen-help.md#table-of-pxie-148x-generation-bitfiles) |
+    | Resource | TDMS File Directory| A directory path containing a file named <font face = "courier new">SI0_LLP_Packets.tdms</font> |
+
+    > Note: The **TDMS File Directory** is a folder selection and the browse dialog shows folders only, not file names. 
+    >
+    > Use the default **TDMS File Directory** value if generating from a TDMS file created with the Create CSI-2 Packet TDMS Files utility's default settings. See the [PXIe-148X Getting Started Example - Basic Generation Tutorial](gse-gen-basic.md#create-tdms-files-for-generation) for details on creating TDMS files.
+
+2. Complete the steps in [PXIe-148X Getting Started Example - Common Acquisition Tutorials](gse-acq-common.md#setting-raw-display-parameters) with the following differences.
+    - Replace all references to <font face = "courier new">SI0</font> with <font face = "courier new">SO0</font>.
+    - When running the Generation Example VI, wait for the **Waiting for Serializer Setup** indicator to illuminate then click **Serializer Setup Complete**.
 
 ## Setting Serial Channel Configurations
 
-## Manually Reading and Writing to GPIO
+This tutorial shows you how to configure the **Serial Channel** tab to generate and display images on multiple serial output channels.
```

Comment:
This section also covers use of the Start Trigger Delay (s) control. Consider updating this intro text to make mention of that since it's tacked onto the end of this section.

Reply:
Done.